src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -95,10 +95,12 @@ public function save(Key $key)
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated
```

Comment:
? this is not how we do deprecation, see https://symfony.com/doc/current/contributing/code/conventions.html#deprecations